presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -234,7 +235,8 @@ private Environment getEnvironment()
                                                 .build()))
                                 .addAll(testArguments)
                                 .addAll(reportsDirOptions(reportsDir))
-                                .build().toArray(new String[0]));
+                                .build().toArray(new String[0]))
+                        .waitingFor(new LogMessageWaitStrategy().withRegEx(".*\\[TestNG] Running.*"));
```

Comment:
Please add a code comment why we want to do so.

Reply:
Sure. I'll open up followup PR